sbws/util/stem.py

```diff
@@ -16,6 +18,9 @@
 stream_building_lock = RLock()
 
 
+TORRC_OPTIONS_CAN_FAIL = OrderedDict({'ConnectionPadding': '0'})
```

Comment:
Let's put all the options in the same place.
You could put this constant in globals.py after TORRC_STARTING_POINT.

Reply:
That was my initial idea (https://trac.torproject.org/projects/tor/ticket/28692#comment:8)